<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StateWindow</class>
 <widget class="QMainWindow" name="StateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1019</width>
    <height>633</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1019</width>
    <height>633</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thermodynamic Simulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(79, 29, 11);
color: rgb(255,255,255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1011</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Thermodynamic States of Cycle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="States">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>941</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pvDiagram_Button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>pv Diagram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TsDiagram_Button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>Ts Diagram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Performance_Button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>Performance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Help_Button">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>550</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
